src/hooks/useZoraCreate.tsx

```diff
@@ -58,8 +61,31 @@ const useZoraCreate = () => {
         account: address!,
       })
       const newParameters = { ...parameters, functionName: 'createContract' }
+      const recipients = getRecipientSortedAddressesAndAllocations(
+        splitArgs.recipients
+      )
+
+      const pullSplitFactory =
+        '0x80f1B766817D04870f115fEBbcCADF8DBF75E017' as Address
+      const splitParameters = {
+        address: pullSplitFactory,
+        abi: pullSplitFactoryAbi as any,
+        functionName: 'createSplit',
+        args: [
+          {
+            recipients: recipients[0],
+            allocations: recipients[1],
+            totalAllocation: BigInt(1000000),
+            distributionIncentive: 0,
+          },
+          zeroAddress,
+          zeroAddress,
+        ],
+        account: address,
+      }
```

Comment:
optional: `splits` code should not live in `zora` lib / hook.
- make a new lib / hook for `splits` code